vm/interpreter/src/default_runtime.rs

```diff
@@ -209,11 +213,21 @@ where
         self.get_actor(&addr).map(|act| act.code)
     }
     fn get_randomness(
-        _personalization: DomainSeparationTag,
-        _rand_epoch: ChainEpoch,
-        _entropy: &[u8],
-    ) -> Randomness {
-        todo!()
+        &self,
+        personalization: DomainSeparationTag,
+        rand_epoch: ChainEpoch,
+        entropy: &[u8],
+    ) -> Result<Randomness, ActorError> {
+        let r = self
+            .rand
+            .get_randomness(&self.store, personalization, rand_epoch, entropy)
+            .map_err(|e| {
+                ActorError::new_fatal(format!("could not get randomness: {}", e.to_string()))
+            })?;
+        let mut array = [0; 32];
+        let bytes = &r[..r.len()];
```

Comment:
did you mean to do array.len()? Irrelevant if you change based on previous comment

Reply:
Oh real nice catch....